hikari/impl/entity_factory.py

```diff
@@ -402,15 +410,19 @@ def deserialize_audit_log(self, payload: data_binding.JSONObject) -> audit_log_m
             changes = []
             if (change_payloads := entry_payload.get("changes")) is not None:
                 for change_payload in change_payloads:
-                    key: typing.Union[audit_log_models.AuditLogChangeKey, str]
-                    key = audit_log_models.AuditLogChangeKey(change_payload["key"])
+                    key: typing.Union[audit_log_models.AuditLogChangeKey, str] = audit_log_models.AuditLogChangeKey(
+                        change_payload["key"]
+                    )
 
                     new_value: typing.Any = change_payload.get("new_value")
                     old_value: typing.Any = change_payload.get("old_value")
                     if value_converter := self._audit_log_entry_converters.get(key):
                         new_value = value_converter(new_value) if new_value is not None else None
                         old_value = value_converter(old_value) if old_value is not None else None
 
+                    elif __debug__ and not isinstance(key, audit_log_models.AuditLogChangeKey):
+                        _LOGGER.debug("Unknown audit log change key found %r", key)
```

Comment:
Also, I think discord sending invalid data should probably be an `info` or so, so that you don't have to be debugging something to complain to discord and get it fixed for everyone... Dunno logging conventions though.

Reply:
This is less invalid data and more just data we haven't implemented the handling for yet which in other places is kept at debug log level right now